lib/exercise_cases.rb

```diff
@@ -1,2 +1,30 @@
 require 'ostruct'
 require 'json'
+
+class ExerciseCase < OpenStruct
+  def name
+    'test_%s' % description.gsub(/[^\w ?!]/, '').gsub(/[- ]/, '_')
```

Comment:
I'm not sure if this is premature:

We will need a `underscore_format(string)`\* method that we can have a bunch of unit tests for so we can keep track of all the weird cases we encounter.

So this line becomes:
`'test_' + underscore_format(description)`

\* name and whether it should be a `String` refinement TBD.